libs/providers/flagd/src/lib/service/grpc/client.ts

```diff
@@ -0,0 +1,21 @@
+import * as grpc from '@grpc/grpc-js';
+import {ServiceClient} from '../../../proto/ts/schema/v1/schema.client'
+import { GrpcTransport } from "@protobuf-ts/grpc-transport";
+
+export default class GRPCClient extends ServiceClient {
```

Comment:
I would recommend avoiding default exports.

Reply:
removed and updated imports